examples/cord_lc/Makefile

```diff
@@ -0,0 +1,42 @@
+# name of your application
+APPLICATION = cord_lc
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+BOARD_INSUFFICIENT_MEMORY := arduino-duemilanove arduino-mega2560 arduino-uno \
+                             chronos hifive1 mega-xplained msb-430 msb-430h  \
+                             nucleo-f030r8 nucleo-l053r8 nucleo-f031k6 \
+                             nucleo-f042k6 nucleo-f303k8 nucleo-f334r8 \
+                             nucleo-l031k6 stm32f0discovery telosb waspmote-pro \
+                             wsn430-v1_3b wsn430-v1_4 z1
+
+USEMODULE += gnrc_netdev_default
+USEMODULE += auto_init_gnrc_netif
+USEMODULE += gnrc_ipv6_default
+USEMODULE += gnrc_icmpv6_echo
+
+USEMODULE += cord_lc
+
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+USEMODULE += fmt
+
+# The default size of GCOAP_PDU_BUF_SIZE can be too small for the response
+# from the RD server. Use a larger value if the client drops response packet
+# from RD server.
+CFLAGS += -DGCOAP_PDU_BUF_SIZE=2048
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+CFLAGS += -DDEVELHELP
```

Comment:
this was changed to `DEVELHELP ?= 1`, a while ago

Reply:
Fixed.